### PR TITLE
Add a /docs/ index page

### PR DESCRIPTION
**Pull request by idrysdale**
Note: I've not added the actual link yet, I'll do that as soon once https://github.com/fluidkeys/www.fluidkeys.com/pull/173 is merged in a separate request.

### CODE REVIEW

File: hugo/content/docs/delete-a-team.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Deleting a team in Fluidkeys"
-description: "Fluidkeys automates key rotation and exchange amongst a team. This guide will get you setup as a team and each member of the team setup with new PGP keys."
+weight: 3
```
Comment (paulfurley) [inline hugo/content/docs/delete-a-team.md]: interesting, what are these `weight` variables? I don't see any reference to them?

File: hugo/layouts/docs/section.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{{ define "title"}}{{ .Title }}{{ end }}
+{{ define "description" }} description: Keep up to date with our progress building Fluidkeys, a way
```
Comment (paulfurley) [inline hugo/layouts/docs/section.html]: Don't think this is supposed to be here!

File: hugo/content/docs/delete-a-team.md (hunk 1 above)
Comment (idrysdale) [reply hugo/content/docs/delete-a-team.md]: they are used to determine where they appear in the `{{ range .Pages }}` ... `{{ end }}` loop in `section.html`.

File: hugo/layouts/docs/section.html (hunk 2 above)
Comment (idrysdale) [reply hugo/layouts/docs/section.html]: Whoops! 👍  I'll drop it and push!

**Comment by idrysdale** [comment]
All updated and good to finish reviewing!

**Pull request merged**